front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -3,38 +3,87 @@ import { connect } from "react-redux";
-                    before:z-10 before:absolute before:-right-5 before:top-1/2 before:w-max before:max-w-xs before:translate-x-full before:-translate-y-1/2 before:rounded-md before:bg-gray before:px-3 before:py-2 before:text-off-white before:invisible before:content-[attr(tooltip)] after:z-10 after:absolute after:-right-[0.8rem] after:top-1/2 after:h-0 after:w-0 after:translate-x-2 after:-translate-y-1/2 after:border-8 after:border-r-gray after:border-l-transparent after:border-b-transparent after:border-t-transparent after:invisible hover:before:visible hover:after:visible
-                    cursor-pointer m-5 w-[5em]"
-                  role="button"
-                  onClick={this.openMainPage}
-                  aria-label="Home Button"
-                  tooltip="Click here to return to homepage"
-                  data-testid="logo-button"
-                >
-                  <Logo />
-                </div>
-                <div className="w-4/5 h-4/5 mx-auto">
-                  <IntentLister intents={analyzedTranscripts[projectId]} />
-                </div>
+      return (
+        <div className="AnalysisPage" data-testid="analysis-page">
+          <aside /* display sidebar if toggled on */
+            aria-hidden={!this.state.showIntentMenu}
+            data-testid="analysis-page-sidebar"
+          >
+            <IntentMenu
+              intents={analyzedTranscripts[projectId]}
+              isOpen={this.state.showIntentMenu}
+              onClickOutside={this.closeIntentMenu}
+            />
+          </aside>
+          <div /* darken screen and hide content from accessibility API if sidebar is toggled on */
+            className={"bg-purple-100 h-screen transition ease-in-out" + (this.state.showIntentMenu ? " brightness-75" : "")}
+            aria-hidden={this.state.showIntentMenu}
+            data-testid="analysis-page-main"
+          >
+            <div className="flex gap-y-10 w-full flex-col h-full">
+              <div /* second line of className is tooltip styling */
+                className="justify-left relative text-center text-sm
+                  before:z-10 before:absolute before:-right-5 before:top-1/2 before:w-max before:max-w-xs before:translate-x-full before:-translate-y-1/2 before:rounded-md before:bg-gray-200 before:px-3 before:py-2 before:text-off-white before:invisible before:content-[attr(tooltip)] after:z-10 after:absolute after:-right-[0.8rem] after:top-1/2 after:h-0 after:w-0 after:translate-x-2 after:-translate-y-1/2 after:border-8 after:border-r-gray-200 after:border-l-transparent after:border-b-transparent after:border-t-transparent after:invisible hover:before:visible hover:after:visible
+                  cursor-pointer m-5 w-[5em]"
+                role="button"
+                onClick={this.openMainPage}
+                aria-label="Home Button"
+                tooltip="Click here to return to homepage"
+                data-testid="logo-button"
```

Comment:
```suggestion
                data-testid="home-button"
```
pre-emptive merge resolution